easybuild/easyconfigs/f/FASTX-Toolkit/FASTX-Toolkit-0.0.14-goolf-1.7.20.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'FASTX-Toolkit'
+version = '0.0.14'
+
+homepage = 'http://hannonlab.cshl.edu/fastx_toolkit/'
+description = """The FASTX-Toolkit is a collection of command line tools for 
+ Short-Reads FASTA/FASTQ files preprocessing."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+altname = name.lower().replace('-','_')
+source_urls = ['https://github.com/agordon/fastx_toolkit/releases/download/%(version)s']
+sources = ['%s-%%(version)s.tar.bz2' % altname]
+
+dependencies = [('libgtextutils', '0.6.1')];
+
+sanity_check_paths = {
+    'files': 
+        ['bin/fastx_%s' % x for x in 
+            ['clipper', 'trimmer', 'quality_stats', 'artifacts_filter', 'reverse_complement',
+            'collapser', 'uncollapser', 'renamer', 'barcode_splitter.pl', 'nucleotide_distribution_graph.sh',
+            'nucleotide_distribution_line_graph.sh']] + 
+        ['bin/fasta_%s' % x for x in 
+            ['clipping_histogram.pl', 'formatter', 'nucleotide_changer']] +
+        ['bin/fastq_%s' % x for x in 
+            ['quality_boxplot_graph.sh', 'quality_converter', 'to_fasta', 'quality_filter', 
+            'quality_trimmer', 'masker']],
+    'dirs': ['.']
```

Comment:
oops, these `'.'` thingies are left-overs from (my?) hackish practice some time ago; better drop/empty that.